addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -323,47 +321,24 @@ def _create_account_move_line(self, move, credit_account_id, debit_account_id, q
             credit_line['debit'] = -diff
         AccountMoveLine.create(debit_line)
         AccountMoveLine.create(credit_line)
-
-        # Create account move lines for quants already out of stock
-        if qty_out > 0:
+        
+        # In anglo-saxon we need to deduct the COGS
+        if self.env.user.company_id.anglo_saxon_accounting and float_compare(extra_cost, 0.0, precision_rounding=self.env.user.company_id.currency_id.rounding) != 0.0:
             debit_line = dict(base_line,
-                              name=(self.name + ": " + str(qty_out) + _(' already out')),
-                              quantity=qty_out,
-                              account_id=already_out_account_id)
+                              name=(self.name + _(' already out')),
+                              quantity=1.0,
+                              account_id=credit_account_id)
             credit_line = dict(base_line,
-                               name=(self.name + ": " + str(qty_out) + _(' already out')),
-                               quantity=qty_out,
-                               account_id=debit_account_id)
-            diff = diff * qty_out / self.quantity
-            if diff > 0:
-                debit_line['debit'] = diff
-                credit_line['credit'] = diff
+                               name=(self.name + _(' already out')),
```

Comment:
Should use string formatting so translations can do the entire thing at once, in case e.g. they need to change the ordering of the phrase.